### ISSUE TITLE
Impress error fail to open PowerPoint file

### ISSUE THREAD

**Issue by edurojasr** [ubuntu/libreoffice#29]
Hi!

I'm try to open a ppt file that I was using before the recent update, now impress fail to open the file and crashes.

- Ubuntu 23.04
- Linux Kernel  Linux 6.2.0-20-generic
- LibreOffice (snap) 7.5.2.2 rev 272
- snapd 2.59.1 rev 18933
- Using Wayland
- Gnome 44

Error message:
```
edu@edu-t14s:~$ snap run libreoffice.impress 

(soffice:14092): GLib-GIO-WARNING **: 21:36:29.803: Error creating IO channel for /proc/self/mountinfo: Permission denied (g-file-error-quark, 2)
/snap/libreoffice/272/lib/libreoffice/program/soffice.bin: symbol lookup error: /snap/libreoffice/272/lib/libreoffice/program/libmergedlo.so: undefined symbol: hb_font_set_synthetic_slant
```

Tested LibreOffice 7.5.2.2 **apt package** (default on Ubuntu 23.04 installation) and this works great, the problem its not present and I can edit and use the ppt file as before. I tested the snap with other ppt file and some works other not and crash with the same error.

Thanks for all the work! Love LibreOffice as snap :hearts:

**Comment by edurojasr** [comment]
Add testing on Ubuntu 22.04.2 LTS the problem persist

**Comment by edurojasr** [comment]
Tested again

 I was using LibreOffice (snap) 7.5.2.2 rev 272 an the moment I use a italic format it crashed, with the fallowing error, so I think the problem is in the snap package of italic font or something

```
/snap/libreoffice/272/lib/libreoffice/program/soffice.bin: symbol lookup error: 
/snap/libreoffice/272/lib/libreoffice/program/libmergedlo.so: undefined symbol: 
hb_font_set_synthetic_slant
```

**Comment by vfazio** [comment]
The 274 release on the candidate channel is not affected at least

**Comment by edurojasr** [comment]
Thanks! Going to check it later

**Comment by edurojasr** [comment]
Closing as fixed in `libreoffice 7.5.3.2 rev 275   channel: latest/stable`
